### ISSUE TITLE
🐞 - Demo should not use any of "tui-" classes other than spacings

### ISSUE THREAD

**Issue by Clindbergh** [taiga-family/taiga-ui#9721]
### Reproduction url

https://taiga-ui.dev/form

### Description

Some of the core layouting building blocks used in [Forms](https://taiga-ui.dev/form) is missing documentation. These elements (like row, skeleton, table) seem to be useful for a lot of other contexts, but some more documentation how they are intended to use them would be helpful.

### Taiga UI version

4.14.0

### Browser Used

Firefox

### OS Used

Windows

**Comment by waterplea** [comment]
It's on purpose, we're phasing out those global styles. I'll change this ticket into "remove deprecated global styles from demo". They were initially released by mistake as our internal set of styles that was a subject to change by another team. We plan to drop them by v5.

**Comment by Clindbergh** [comment]
Thanks for the quick reply.

I encountered another issue and I was wondering if this is related. `tuiSurfaceLayer` is used in some demos (like selectable CardMedium), but [I cannot find any definitions for it](https://github.com/search?q=org%3Ataiga-family%20tuiSurfaceLayer&type=code).

Also: Is the website open source? It would be helpful to understand how the demos work.

**Comment by Clindbergh** [comment]
Ah, that explains it. I tried to use [a selectable card](https://taiga-ui.dev/layout/card-medium#selectable), but had problems to set it up. I assumed this was due to the missing `tuiSurfaceLayer` directive (or what I thought would be a directive).

Good to know about the website. Thanks for the quick response and elaboration.

---

Regarding my original request:

> They were initially released by mistake as our internal set of styles that was a subject to change by another team. We plan to drop them by v5.

The [form](https://taiga-ui.dev/form#form) example was actually pretty helpful. It was the only sample that displayed a full form demo where a lot of capabilities are shown. Since you're planning to drop these classes in v5: Do you recommend any other framework for layouting (perhaps known to work well and with similar conventions as taiga)?

**Comment by waterplea** [comment]
A lot of people use Tailwind with Taiga UI and like it. I don't use layout libraries myself as I prefer to style all the pages myself.

**Comment by shiv9604** [comment]
Hi @waterplea,
Just wanted to confirm before starting on it, Are we concerned about only Classes mentioned [here](https://taiga-ui.dev/form) or we are  considering removing other classes as well for example `tui-text_h6, tui-list_*, tui-island_* being used in demo. I would like to take on this & you can assign the same.

**Comment by waterplea** [comment]
Ideally we want to remove everything from this package:
https://github.com/taiga-family/taiga-ui/blob/v3.x/projects/styles/taiga-ui-global.less
It doesn't have to be a single PR, if you can remove classes from form page — that would already be helpful. We are moving away from those global classes to specific layout components. For example tui-island classes interfaces are now supposed to be handled by `CardLarge` component.

**Comment by shiv9604** [comment]
That was helpful & descriptive. 

In the documentation, as of now we just need to show the notification for its depreciation as displayed [here](https://taiga-ui.dev/components/island) or do we need to fully integrate it in docs with new layout component & change the docs accordingly?

**Comment by waterplea** [comment]
> That was helpful & descriptive. 
> 
> In the documentation, as of now we just need to show the notification for its depreciation as displayed [here](https://taiga-ui.dev/components/island) or do we need to fully integrate it in docs with new layout component & change the docs accordingly? 
> 

You can just start with getting rid of those classes and replacing them with layout components or in-place styles so it looks right. At this point it's more important to drop them from examples so new people will not start using them.

**Comment by shiv9604** [comment]
Thank you for the clarity!

**Comment by shiv9604** [comment]
HI @waterplea, I am done with the `tui-form`, and `tui-table` deprecated classes migration. Are the `tui-list` specs ready from the design team, so I can raise the PR make necessary modifications in my existing implementation if required & increase the PR which might close this issue completely?
